packages/core/src/Synchronizer.ts

```diff
@@ -127,6 +127,11 @@ export class Synchronizer extends EventEmitter {
      * How many blocks the synchronizer will query on each poll. Default: `100000`
      */
     public blockRate: number = 10000
+    /**
+     * The genesis block of the provider environment
+     * @dev Allow passing the genesis block as the starting block for querying events
```

Comment:
Actually this is not like solidity to write `@dev`
It is in typedoc format, you can check the reserved word and usage here: https://typedoc.org/guides/overview/

I prefer just written for developers
so it will be like
```ts
/**
 * Allow passing the genesis block as the starting block for querying events
 */
```

Reply:
Thank you for pointing out this issue. It is great to learn this.